changelog/1.16.2.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "1.16.2"
+description: "Released on 03/09/2021"
+---
+
+### Breaking Changes ❗
+
+There are no breaking changes in 1.16.2.
+
+### Features ✨
+
+- feat: Add environments.nodeSelector to the Helm chart, which enables selecting
+  specific nodes for environment pods to be scheduled on.
+
```

Comment:
would make more sense prefixed as `helm` since this is already under a `Features` heading